flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -124,7 +126,7 @@ public HiveCatalog(String catalogName, String defaultDatabase, HiveConf hiveConf
 		LOG.info("Created HiveCatalog '{}'", catalogName);
 	}
 
-	private static HiveConf getHiveConf(String hiveMetastoreURI) {
+	public static HiveConf getHiveConf(String hiveMetastoreURI) {
```

Comment:
Instead of exposing this method, maybe we should move the logic to the caller instead. After all, the logic is simple enough and HiveConf instance is all HiveCatalog needs in order to connect to HMS properly. How to get HiveConf is up to the caller.